src/main/java/gov/usds/case_issues/db/model/TroubleCase.java

```diff
@@ -39,16 +39,29 @@
 @NamedNativeQueries({
 	@NamedNativeQuery(
 		name = "snoozed",
-		query = "SELECT * from " + TroubleCase.CASE_DTO_CTE
-			  + "WHERE last_snooze_end >= CURRENT_TIMESTAMP "
-			  + "ORDER BY last_snooze_end ASC, case_creation ASC, internal_id ASC",
+		query = "SELECT * FROM " + TroubleCase.CASE_DTO_CTE
```

Comment:
I think we might want to make a snoozed-case CTE as well, just to avoid repetition.